content/resources/core/sybase_conf.md

```diff
@@ -2,72 +2,6 @@
 title = "sybase_conf resource"
 draft = false
 
-platform = "os"
-
-[menu.resources]
-    title = "sybase_conf"
-    identifier = "resources/core/sybase_conf.md sybase_conf resource"
-    parent = "resources/core"
 +++
 
-Use the `sybase_conf` Chef InSpec audit resource to test the configuration of an SAP Adaptive Server Enterprise (ASE) database.
-
-## Availability
-
-### Install
-
-{{< readfile file="content/reusable/md/inspec_installation.md" >}}
-
-### Requirements
-
-The `isql` command line tool must be installed on the target system.
-
-You must have access to a database user that has access to the `sa` role on the `master` database.
-
-## Syntax
-
-A `sybase_conf` resource block declares the configuration item name, server, and password to use.
-
-```ruby
-describe sybase_session('config item', server: 'SYBASE', password: 'PASSWORD') do
-  its('run_value') { should cmp 'EXPECTED' }
-  its('config_value') { should cmp 'EXPECTED' }
-end
-```
-
-where:
-
-- `sybase_conf` declares a configuration item, server, and password with permission to run `sp_configure`.
-- `its('run_value') { should cmp 'expected' }` compares the current running value of the configuration item against an expected value.
-- `its('config_value') { should cmp 'expected' }` compares the saved value of the configuration item against an expected value.
-
-### Optional Parameters
-
-The `sybase_conf` resource is based on the `sybase_session` resource and accepts all parameters that `sybase_session` accepts, including optional parameters `username`, `database`, `sybase_home`, and `bin`.
-
-In particular:
-
-#### `database`
-
-Defaults to `master`.
-
-#### `username`
-
-Defaults to `sa`.
-
-## Examples
-
-The following examples show how to use this Chef InSpec audit resource.
-
-### Test for max memory configuration
-
-```ruby
-describe sybase_session('max memory', server: 'SYBASE', password: 'PASSWORD') do
-  its('run_value') { should cmp 180224 }
-  its('config_value') { should cmp 180224 }
-end
-```
-
-## Matchers
-
-{{< readfile file="content/reusable/md/inspec_matchers_link.md" >}}
+{{< readfile file="content/reusable/md/modularized_resource.md" >}}
```

Comment:
Can we remove the sybase, opa and ibmdb2 all resources from this list for now as those won't be part of this release. Debdan will be giving content to address breaking of this resources in the release